HubbleHW.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>I think you chose the wrong parameter of your fit. a=55.99 seems to be the right one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Hubble/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Just ignore the comment: see eg (https://en.wikipedia.org/wiki/Hubble%27s_law#Measured_values_of_the_Hubble_constant)